catalogue/webapp/pages/works.js

```diff
@@ -41,7 +38,15 @@ export const Works = ({
   handleSubmit,
   version
 }: ComponentProps) => (
-  <Fragment>
+  <PageLayout
+    title={`${query ? `${query} | ` : ''}Catalogue search`}
```

Comment:
@hthair the page titles on `/works` are going to change with this PR. What used to be

`Image catalogue search${query ? `: ${query}` : ''}`

will become

`${query ? `${query} | ` : ''}Catalogue search`

e.g.

`Image catalogue search: eggs` will become `eggs | Catalogue search`